dev/tasks/docker-tests/azure.linux.yml

```diff
@@ -49,5 +51,16 @@ jobs:
   - script: pip install -e arrow/dev/archery[docker]
     displayName: Setup Archery
 
-  - script: archery docker run -e SETUPTOOLS_SCM_PRETEND_VERSION="{{ arrow.no_rc_version }}" {{ run }}
+  - script: |
+      archery docker run -e SETUPTOOLS_SCM_PRETEND_VERSION="{{ arrow.no_rc_version }}" {{ run }}
```

Comment:
@kou we need to pass `ARROW_DOCS_VERSION` instead of `SETUPTOOLS_SCM_PRETEND_VERSION`, see https://github.com/apache/arrow/blob/master/dev/release/post-09-docs.sh#L48

Reply:
OK!
I'll specify `ARROW_DOCS_VERSION` in `run`.

Reply:
I'll do this in https://github.com/apache/arrow/pull/11861f .